tests/data/transcription/est00.txt

```diff
@@ -0,0 +1,5 @@
+0.120   0.290   225.000
```

Comment:
No, I have the ground truth, but no algorithm and no algorithm output. Due to the time difference Li Su and I are pretty much limited to an email a day, but he's been very responsive so far, so with a bit of luck I might get his alg's output from even this evening (EST).

I could search to see if there's code I could run, but tbh give everything else I need to do I'd rather keep it simple and get the alg output from someone directly.